packages/block-editor/src/components/block-breadcrumb/breadcrumb-button.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * WordPress dependencies
+ */
+import { PrimitiveButton, useTheme } from '@wordpress/components';
+
+export default function BlockBreadcrumbButton( { children, ...props } ) {
+	const theme = useTheme();
+	const additionalStyles = [
```

Comment:
Is this for things that don't translate to mobile?

Reply:
Since we don't want to expose the `css` prop in public API, there is no way to set styles for i.e` hover`, `focus` states. Our first idea was to use props like `hoverStyle`,` activeStyle` ... but these styles can be mixed so we will end with `hoverFocusActiveEnabledStyle` and that would be hard to use. So we decided to add `additionalStyles` prop where we can set styles for different states.

Reply:
So these work for mobile too?

Reply:
We don't have all these states on mobile, so most of them will be omitted there.